tests/test_aspa.c

```diff
@@ -7,351 +7,412 @@
 * Website; http://rtrlib.realmv6.org/
 */
 
-
-#include "rtrlib/aspa/aspa_array/aspa_array.h"
 #include "rtrlib/aspa/aspa.h"
+#include "rtrlib/aspa/aspa_array/aspa_array.h"
 #include "rtrlib/aspa/aspa_private.h"
 #include "rtrlib/lib/alloc_utils_private.h"
 #include "rtrlib/lib/convert_byte_order_private.h"
+#include "rtrlib/rtr/packets_private.h"
 #include "rtrlib/rtr/rtr_pdus.h"
 #include "rtrlib/transport/transport.h"
-#include "rtrlib/rtr/packets_private.h"
 
 #include <assert.h>
 #include <stdbool.h>
 #include <stdint.h>
 #include <stdio.h>
 #include <string.h>
 
-/*
-static void update_cb(struct aspa_table *t __attribute__((unused)), const struct aspa_record rec, const struct rtr_socket *socket, const bool added)
+char *data;
+size_t data_size = 0;
+size_t data_index = 0;
+
+#define c16(X) lrtr_convert_short(TO_HOST_HOST_BYTE_ORDER, X)
+#define c32(X) lrtr_convert_long(TO_HOST_HOST_BYTE_ORDER, X)
```

Comment:
This should should be named something more descriptive

Reply:
please give suggestions. i wanted to keep it very short: convert 16 bit / convert 32 bit

Reply:
maybe we could call it `HOST_BYTES`